python3/primes.py

```diff
@@ -1,13 +1,12 @@
-def is_prime(num):
-    """Returns True if the number is prime
-    else False."""
-    if num == 0 or num == 1:
-        return False
-    for x in range(2, num):
-        if num % x == 0:
-            return False
-    else:
-        return True
-
-result = filter(is_prime, range(1, 20000))
-
+primes = [2]
+for i in range(3, 100000): # Incorrect range!
+    is_prime = True
+    for j in range(0, len(primes)):
```

Comment:
Yes, it will traverse the list in order if you append the list, which it does already.